.github/workflows/pypi.yaml

```diff
@@ -23,7 +23,8 @@ jobs:
     - name: Set version from tag
       run: |
         ETL_VERSION=$(echo $GITHUB_REF_NAME | sed 's/^v//')
-        sed -i "s/0\.0\.0/$ETL_VERSION/" cumulus_etl/__init__.py
+        sed -i "s/1\!0\.0\.0/$ETL_VERSION/w changes.txt" cumulus_etl/__init__.py
+        [ -s changes.txt ] || exit 1  # validate that we successfully set the version
```

Comment:
While I was here, I also tweaked the release version code to match what I did in data-metrics:
- Use an epoch for the development/git version, so that it will always be higher than the releases
- Confirm that we did correctly regex in the release pipeline, to try and avoid a future mixup